gettsim/transfers/grundrente.py

```diff
@@ -0,0 +1,397 @@
+import pandas as pd
+
+from gettsim.piecewise_functions import piecewise_polynomial
+from gettsim.shared import add_rounding_spec
+from gettsim.typing import BoolSeries
+from gettsim.typing import FloatSeries
+from gettsim.typing import IntSeries
+
+
+@add_rounding_spec(params_key="ges_rentenv")
+def grundr_zuschlag_m(
+    grundr_zuschlag_vor_eink_anr_m: FloatSeries, grundr_zuschlag_eink_m: FloatSeries
+) -> FloatSeries:
+    """Calculate Grundrentenzuschlag (additional monthly pensions payments
+    resulting from Grundrente)
+
+    Parameters
+    ----------
+    grundr_zuschlag_vor_eink_anr_m
+        See :func:`grundr_zuschlag_vor_eink_anr_m`.
+    grundr_zuschlag_eink_m
+        See :func:`grundr_zuschlag_eink_m`.
+
+    Returns
+    -------
+
+    """
+    out = grundr_zuschlag_vor_eink_anr_m - grundr_zuschlag_eink_m
+    return out.clip(lower=0)
+
+
+@add_rounding_spec(params_key="ges_rentenv")
+def grundr_zuschlag_eink_m(
+    gemeinsam_veranlagt: BoolSeries,
+    eink_excl_grundr_zuschlag_m_tu: FloatSeries,
```

Comment:
Great idea!

Does `ges_rente_inkl_grundr_m` -> `ges_rente_nach_grundr_m` also make sense?

Reply:
Oh, I only realise now what is going on here, I thought it would be actually something intermediate before `ges_rente_m` comes out. 

Please swap to "bis_2020" and "ab_2021" as in other cases where interfaces change over time. Will have a better mechanism for that soon!

Reply:
It's a mixture of both: after 2021 you could still be interested in `ges_rente_vor_grundr_m`

Reply:
In that respect, I like the current names more. But would be also open to change it if you prefer.

Reply:
Fair enough. Shows we really need a new way to work with those changes over time...

Reply:
Do as you see fit, also below.